docs/source/install/guide.rst

```diff
@@ -21,31 +18,62 @@ Install Dependencies
 
 Install MongoDB, RabbitMQ, and PostgreSQL.
 
+  .. rst-class:: ubuntu
+
+    Ubuntu/Debian
+
   .. code-block:: bash
 
-    sudo apt-get update
+    sudo apt get update
     sudo apt-get install -y mongodb-server rabbitmq-server postgresql
 
+  .. rst-class:: centos
+
+    CentOS/RHEL
```

Comment:
Looks like the change is to include both rpm/deb instructions in one big guide.

I worked on testing new packages under `RHEL7/CentOS7` and draft with minimal installation instructions is: https://gist.github.com/armab/2ab4d2ccc592db42243

Some points to avoid this merge if possible: 
- There are already slightly different instructions in logic for `rpm`, comparing to `deb`. Like for `el7` you need additional commands, which are not required for `ubuntu`.
- There is even a chance that instructions for `el6` comparing to `el7` will slightly differ. I didn't tested `el6`, focusing on `el7` first, but I suspect where could be differences. That means - more "branching" in one doc, where you need to cover more special cases.
- A lot of text makes instructions always looking harder both in complexity and readability/usability.

Yes, it would be less easy for us to manage and keep in sync repeating/duplicated pieces, but I believe it's overall easier for user with separated docs, keeping in mind that the goal is successful and more painless/easy installation.